examples/apps/ai_remote_infer_app/__main__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021-2025 MONAI Consortium
```

Comment:
Should we set the copyright year to 2025 for the new files?

Reply:
Good point. Was debating to have year range or just the year when the new file is introduced. Will update to have just the year when introduced.